src/core.cpp

```diff
@@ -1546,15 +1546,19 @@ static int gpu_monitoring([[maybe_unused]] Data &data)
 			case GpuDrv::GPUDRV_AMDGPU:
 			{
 				const std::string amdgpu_gpu_busy_file = (!card.drm_path.empty()) ? card.drm_path + "/device/gpu_busy_percent" : std::string();
+				const std::string amdgpu_mem_clock     = card.hwmon_path + "/freq2_input";
+				const std::string amdgpu_power1_average= card.hwmon_path + "/power1_average";
 				MSG_DEBUG("gpu_monitoring: amdgpu: amdgpu_gpu_busy_file=%s", amdgpu_gpu_busy_file.c_str());
 				FOPEN_TO_ITEM(vbios_version, "%s/device/vbios_version",        card.drm_path.c_str());
 				// temperature obtained above
 				FOPEN_TO_ITEM(usage,         "%s", amdgpu_gpu_busy_file.c_str()); // Linux 4.19+ (22 October 2018)
 				FOPEN_TO_ITEM(core_voltage,  "%s/in0_input",                   card.hwmon_path.c_str());
-				FOPEN_TO_ITEM(power_avg,     "%s/power1_average",              card.hwmon_path.c_str());
+				FOPEN_TO_ITEM(power_avg,     "%s/power1_input",              card.hwmon_path.c_str());
```

Comment:
What if `power1_input` file does not exist? That is the case for my graphic card.

Reply:
you don't have power1_input, but you do have power1_average ? very interesting...